test/test_sign_runtime_policy.py

```diff
@@ -0,0 +1,194 @@
+"""
+SPDX-License-Identifier: Apache-2.0
+Copyright 2024 Red Hat, Inc.
+"""
+
+import argparse
+import os
+import tempfile
+import unittest
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import ec
+from cryptography.hazmat.primitives.serialization import load_pem_private_key
+
+from keylime.cert_utils import is_x509_cert
+from keylime.policy import sign_runtime_policy
+from keylime.policy.logger import Logger
+from keylime.signing import verify_dsse_envelope
+
+DATA_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "data", "sign-runtime-policy"))
+EC_PRIVKEY = os.path.join(DATA_DIR, "ec-p521-private.pem")
+EC_PUBKEY = os.path.join(DATA_DIR, "ec-p521-public.pem")
+RSA_PRIVKEY = os.path.join(DATA_DIR, "rsa-4096-private.pem")
+
+POLICY = os.path.join(DATA_DIR, "runtime-policy.json")
+POLICY_EMPTY = os.path.join(DATA_DIR, "runtime-policy-empty.json")
+POLICY_BOGUS = os.path.join(DATA_DIR, "runtime-policy-bogus.json")
+
+
+class SignRuntimePolicy_Test(unittest.TestCase):
+    def test__get_signer(self):
```

Comment:
s/test__get_signer/test_get_signer/

Reply:
This was because the method name is `_get_signer()`, but I can remove one of these underscores, if you feel strongly about it.

Reply:
No need for the change